drivers/SmartThings/matter-switch/profiles/light-color-level.yml

```diff
@@ -15,7 +15,4 @@ components:
   - id: refresh
     version: 1
   categories:
-  - name: Light
-metadata:
-  mnmn: SmartThingsEdge
-  vid: generic-rgbw-color-bulb-2580K-7050K
+  - name: Light
```

Comment:
Missing newline to end the file

Reply:
Fixed in https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1220/commits/f04cf2ec2c69373a72d337e4376c0565baa95f56